src/Libraries/CoreNodeModelsWpf/Controls/ElementSelectionControl.xaml

```diff
@@ -7,7 +7,75 @@
+        <ResourceDictionary>
+            <converters:SelectionButtonContentConverter x:Key="SelectionButtonContentConverter" />
+            <Style x:Key="SelectionButtonStyle" TargetType="Button">
+                <Setter Property="HorizontalAlignment" Value="Right" />
+                <Setter Property="VerticalAlignment" Value="Center" />
+                <Setter Property="Margin" Value="15,0,0,0" />
+                <Setter Property="Template">
+                    <Setter.Value>
+                        <ControlTemplate TargetType="Button">
+                            <Grid>
+                                <Border x:Name="mouseOverBorder"                                
+                                    Margin="2"
+                                    BorderThickness="2"
+                                    CornerRadius="1"
+                                    Visibility="Hidden">
+                                    <Border.BorderBrush>
+                                        <SolidColorBrush Color="#FFFFFF" Opacity="0.15"/>
+                                    </Border.BorderBrush>
+                                </Border >
+                                <Border x:Name="mousePressedBorder"                                
+                                    Margin="2"
+                                    BorderThickness="2"
+                                    CornerRadius="1"
+                                    Visibility="Hidden">
+                                    <Border.BorderBrush>
+                                        <SolidColorBrush Color="#38ABDF" Opacity="0.35"/>
+                                    </Border.BorderBrush>
+                                </Border >
+                                <Border x:Name="border"
+                                    Margin="4"
+                                    Height="36px"
+                                    Padding="8,0,8,0"
+                                    BorderThickness="1"                                                                                                                             
+                                    Background="{TemplateBinding Background}"
+                                CornerRadius="1">
+                                    <Border.BorderBrush>
+                                        <SolidColorBrush Color="#FFFFFF" Opacity="0.5"/>
+                                    </Border.BorderBrush>
+
+                                    <TextBlock x:Name="textBlock"
+                                       HorizontalAlignment="Center"
+                                       Margin="0,1,0,0"
+                                       VerticalAlignment="Center"
+                                       FontFamily="{StaticResource ArtifaktElementRegular}"
+                                       FontSize="14px"
+                                       Foreground="#FFFFFF"
```

Comment:
for all the colors defined here, it would be better to use the colors that are already defined in the dynamo modern and other resource dictionaries so that changing the color in the single place will update all controls, for example if we want to make themeing easy, or support different color schemes for accessibility.

Reply:
Done, thanks for the advice.